client/rpc/src/eth/fee.rs

```diff
@@ -90,7 +90,7 @@ where
 			};
 			// Highest and lowest block number within the requested range.
 			let highest = UniqueSaturatedInto::<u64>::unique_saturated_into(number);
-			let lowest = highest.saturating_sub(block_count);
+			let lowest = highest.saturating_sub(block_count - 1);
```

Comment:
Need to handle the case when `block_count` is `0`.

Reply:
Thank you, just changed it